yeonjae/programmers-1/포켓몬.js

```diff
@@ -0,0 +1,10 @@
+function solution(nums) {
+  const arr = new Set(nums);
+  const limitNum = Math.floor(nums.length / 2);
+
+  if (limitNum > arr.size) {
+    return arr.size;
+  } else {
+    return limitNum;
+  }
+}
```

Comment:
```suggestion
function solution(nums) {
  const setNums = new Set(nums);
  const limitNum = Math.floor(nums.length / 2);
    
  if (limitNum > setNums.size) {
    return setNums.size;
  }
  return limitNum;
}
```
- set 을 이용해 중복을 제거해서 결과를 도출한 풀이 잘봤습니다. 아쉬운 점이라면 set 자료구조를 사용하기 때문에 변수명이 arr인건 아쉽네요 setNums로 변경하였습니다.
- 그리고 else 문은 불 필요한 블록문이라 제거하였습니다 🙇‍♂️
- 다른분들과 다르게 나누기 연산을 할 때 Math.floor를 해서 소수점을 제거해주신 점을 좋았습니다 :)

Reply:
넵 변수명 좀 더 신경써서 만들어볻록하겠습니다!